src/relative-time-element.js

```diff
@@ -4,6 +4,21 @@ import RelativeTime from './relative-time'
 import ExtendedTimeElement from './extended-time-element'
 import {localeFromElement} from './utils'
 
+// Internal: Array tracking all elements attached to the document that need
+// to be updated every minute.
+const nowElements = new Set()
+
+// Internal: Timer ID for `updateNowElements` interval.
+let updateNowElementsId
+
+// Internal: Install a timer to refresh all attached relative-time elements every
+// minute.
+function updateNowElements() {
+  clearTimeout(updateNowElementsId)
+  for (const el of nowElements) el.textContent = el.getFormattedDate() || ''
```

Comment:
This is perhaps not related to this change, per se, but if I understand correctly, anytime a new `<relative-time>` element appears, we'll run `updateNowElements()` since it's called from ` connectedCallback()`. This seems wasteful since we'll go update all other elements on the page and just the new one. Am I understanding that correctly?

Maybe related, the parent runs https://github.com/github/time-elements/blob/4414cbacdb1e0cf95b7e99960897bb0c55736323/src/extended-time-element.ts#L16-L19 as part of `connectedCallback` so do we even need to worry about setting the initial value from this class?

Reply:
This is a valid point! We also have changed the semantics of this element because of this change!